dev-python/mongoengine/mongoengine-0.25.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="A Python Object-Document-Mapper for working with MongoDB"
+HOMEPAGE="
+	https://github.com/MongoEngine/mongoengine/
+	http://mongoengine.org/
+	https://pypi.org/project/mongoengine/
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-python/pymongo[${PYTHON_USEDEP}]
+"
+
+distutils_enable_tests pytest
```

Comment:
Did the tests pass for you? I mean, before you switched to pypi sdist because it has no tests. But after switching to GitHub archive I get tons of test failures, and now I recall precisely why we did remove this package from Gentoo.

Reply:
I *think* I had the test passing back then, but not sure. I removed pytest-randomly and this dependency as well.

Reply:
Ah, cool. I think some other package wanted factory-boy at some point but if we can avoid the problem, that works for me.